project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   val slf4jVersion = "1.7.5"
   val awsVersion = "1.11.7"
   val faciaVersion = "2.0.2"
-  val capiModelsVersion = "9.24"
+  val capiModelsVersion = "10.1"
```

Comment:
@mchv says the `content-api-models-scala` and `content-api-models-json` are included as a transitive dependency of the `content-api-client`, so I don't think we need to import these specifically, so we should be able to remove these